pkgs/development/tools/misc/hydra/unstable.nix

```diff
@@ -10,23 +10,26 @@
 , unzip
 , pkg-config
 , sqlite
-, libpqxx_6
-, top-git
+, libpqxx
+, gitAndTools
```

Comment:
gitAndTools is a deprecated collection of aliases, we should be using `top-git` from the top level:
```
$ nix-instantiate --arg config '{allowAliases=false;}' -A hydra_unstable
error: Function called without required argument "gitAndTools" at /home/linus/nixpkgs/master/pkgs/development/tools/misc/hydra/unstable.nix:14
```